### PR TITLE
Feature/quick fix

### PR DESCRIPTION
**Pull request by Freaxed**

### CODE REVIEW

**Review by jackburton79 — APPROVED** [review]
Tried: works great!

**Comment by Freaxed** [comment]
Not sure about the menu labe (why Quick fix?) moreover I world like to add the change description proposed by the compiler (like 'remove unused include')

**Comment by jackburton79** [comment]
> Not sure about the menu labe (why Quick fix?) moreover I world like to add the change description proposed by the compiler (like 'remove unused include')

That would be better, definitely

**Comment by nexus6-haiku** [comment]
If a quick-fix is not available I'd suggest either not showing the menu item or disabling it and give it a label like "No quick-fix available".

File: src/ui/ProblemsPanel.cpp [hunk 1]
```diff
@@ -40,21 +43,33 @@ ProblemsPanel::ProblemsPanel(BTabView* tabView): BColumnListView(ProblemLabel,
 									, fTabView(tabView)
 
 {
-	AddColumn(new BStringColumn(B_TRANSLATE("Category"),
+	BPopUpMenu*	menu = new BPopUpMenu("_popUp");
+	menu->AddItem(new BMenuItem("Test", nullptr));
```
Comment (nexus6-haiku) [inline src/ui/ProblemsPanel.cpp:47]: Is it a leftover?

**Comment by Freaxed** [comment]
Yes it is! Thank you!


Il giorno mar 21 nov 2023 alle ore 10:39 Davide Alfano <
***@***.***> ha scritto:

**Pull request merged**